tensorbay/opendataset/VOC2012Detection/loader.py

```diff
@@ -60,29 +60,29 @@ def VOC2012Detection(path: str) -> Dataset:
     for segment_name in _SEGMENT_NAMES:
         segment = dataset.create_segment(segment_name)
         with open(os.path.join(main_path, f"{segment_name}.txt"), encoding="utf-8") as fp:
-            for filename in fp:
-                segment.append(_get_data(filename.rstrip(), image_path, annotation_path))
+            for stem in fp:
+                segment.append(_get_data(stem.rstrip(), image_path, annotation_path))
     return dataset
 
 
-def _get_data(filename: str, image_path: str, annotation_path: str) -> Data:
+def _get_data(stem: str, image_path: str, annotation_path: str) -> Data:
     """Get all information of the datum corresponding to filename.
 
     Arguments:
-        filename: The filename of the data.
+        stem: The filename of the data.
```

Comment:
The description of `stem` is `filename` is weird, maybe `The stem of the data.`?